README.md

```diff
@@ -71,11 +70,11 @@ class UsageExample
 
         $config = [
             'headers' => [
-                'Authorization' => 'Bearer '.$accessToken
+                'Authorization' => $accessToken
```

Comment:
```suggestion
                'Authorization' => 'Bearer '.$accessToken
```
Would the request be successful without the bearer prefix?

Reply:
At least I couldn't get it to work with the keyword. 🤷‍♂️

I read the tip to leave out the `Bearer` somewhere at stackoverflow.com. But I can't find the source at the moment. 🙈

It worked for me after removing the keyword!

Reply:
Interesting. I'm able to get it working with the "prefix" and without. I'll resolve this for now.